hazelcast/src/test/java/com/hazelcast/osgi/HazelcastOSGiServiceTest.java

```diff
@@ -132,6 +206,64 @@ public void defaultInstanceExistWhenItIsSpecified() throws BundleException {
     }
 
     @Test
+    public void serviceCouldNotBeActivatedWhenThereIsExceptionWhileRegisteringDefaultInstance() throws BundleException {
+        String propValue = System.getProperty(HazelcastOSGiService.HAZELCAST_OSGI_START);
+        TestBundle testBundle = null;
+        try {
+            System.setProperty(HazelcastOSGiService.HAZELCAST_OSGI_START, "true");
+
+            TestBundleRegisterDeregisterListener registerDeregisterListener =
+                    new TestBundleRegisterDeregisterListener(HazelcastInstance.class.getName());
+            registerDeregisterListener.throwExceptionOnRegister = true;
+
+            testBundle = new TestBundle(registerDeregisterListener);
+            try {
+                testBundle.start();
+                fail("OSGI service could not be activated because of exception while registering default instance. " +
+                     "It is expected to get `IllegalStateException` here!");
+            } catch (IllegalStateException e) {
```

Comment:
It can be more readable to use `expected` in the `@Test` annotation instead of `fail`+ `empty catch block`